+sw_tests/+unit_tests/unittest_spinw_addatom.m

```diff
@@ -0,0 +1,150 @@
+classdef unittest_spinw_addatom < sw_tests.unit_tests.unittest_super
+
+    properties
+        swobj = [];
+        abstol = 10*eps
```

Comment:
I've seen use of 10*eps in a few places, thinking about it maybe this should be the default abstol in `verify_val`? As we already have `1e-10` for the relative tolerance. Feel free to update it.

Reply:
Good idea, will do!

Reply:
Ah with the way it's currently set out, the default value of `abs_tol = 0` in `verify_val` means it only tests for the relative tolerance being met, if we add a non-zero default value for `abs_tol` then we will compare both the relative tolerance and absolute (even if only 4 arguments are provided) . Also atm if you want to supply `abs_tol` you must specify a `rel_tol` - perhaps `verify_val` should accept key word arguments/name-value pairs so that we can easily supply either `abs_tol`, `rel_tol` or both. What do you think?